crates/solana-listener/src/component.rs

```diff
@@ -301,4 +306,109 @@ mod tests {
             );
         }
     }
+
+    #[test_log::test(tokio::test(flavor = "current_thread"))]
+    async fn can_force_last_processed_signature_via_env_var() {
```

Comment:
I still need to make this test to work.

Reply:
random note: I believe that the simplest approach to do this:

inside the  `processor()` function just read the env variable and update the `state` from there. Then you won't have to add additional logic in the realtime scanner function

Reply:
Fixed in latest commits recheck :)